src/store/plugins/veeValidate.js

```diff
@@ -54,6 +55,7 @@ configure({
       max_redeem: ({ rule }) => tg('validation.maxRedeem', [rule.params[0]]),
       max_value_vault: ({ rule }) => tg('validation.maxValueVault', [rule.params[0]]),
       max_len: ({ rule }) => tg('validation.maxLength', [rule.params[0]]),
+      hex: () => tg('validation.hexValue'),
```

Comment:
Can you please return the translated message directly in the `defineRule` scope? Please see the new structure of `veeValidate` file that will be soon merged: https://github.com/superhero-com/superhero-wallet/pull/2485/files#diff-012d1e7b17f11659baed376635edd5f2a3482db91cba378b29c82839f37e52fb
The idea is to have the rule and the message in one place.